### DISCUSSION TITLE
Inline completions as code completions

### DISCUSSION THREAD

**Discussion by Folamh** [zed-industries/zed#27393] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

I would like if we could make inline completions appear in the code completion box. There is an example of this for nvim through https://github.com/zbirenbaum/copilot-cmp. I get less distracted by the suggestions and I can get both lsp suggestions and suggestions from copilot. 

### If applicable, add mockups / screenshots to help present your vision of the feature

![image](https://github.com/user-attachments/assets/47b3af72-1177-4eed-a944-bfc901eb02b8)

**Comment by whazor** [comment]
Please consider utilizing the [LSP Inline Completion Request](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.18/specification/#textDocument_inlineCompletion) to allow multiple extension to provide this functionality.

**Comment by notpeter** [comment]
@whazor I've opened a separate issue for tracking that:
- https://github.com/zed-industries/zed/issues/27392

We partially support this with Zed Edit Predictions:

<img width="550" alt="Image" src="https://github.com/user-attachments/assets/76e6fd7f-6a40-44b8-a56f-e6f603a5c687" />
